frontend/src/component/Home/Home.styl

```diff
@@ -55,3 +55,8 @@
 
   .homepagecard
     width 100%
+
+@media (max-height: max-xs)
+
+  .homepagecard
+    height 100%
```

Comment:
please just add a new line at the end of file.

It's a good developer habit to always add a new line at end of files.

Most code editor have an option to automatically add it